SeleniumDemoQA/SeleniumDemoQA/WebDriverExtensions.cs

```diff
@@ -87,6 +87,27 @@ public static void WaitForElementEnabled(this IWebDriver driver, By by)
             wait.Until(d => d.FindElement(by).Displayed && d.FindElement(by).Enabled);
         }
 
+        public static void ClickOnTableActionElement(this IWebDriver driver, By selector)
+        {
+            var element = driver.GetElementBy(selector);
+            driver.ScrollTo(element);
+
+            // Перевірка на видимість і доступність елемента
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(5));
```

Comment:
these 2 lines should be in separate method WaitUntilElementEnabled
You don't need to scroll to element first in order to click it, so in some Page method you need to write like
_driver.WaitUntilElementEnabled(selector);
_driver.ClickWithJs(selector);

Reply:
Ok, understood, separated, Thanks..!)